tests/e2e/transaction/transactionSimulation.test.ts

```diff
@@ -1,17 +1,29 @@
-import { Account, U64, SigningSchemeInput } from "../../../src";
+import { Account, U64, SigningSchemeInput, InputEntryFunctionData } from "../../../src";
 import { longTestTimeout } from "../../unit/helper";
 import { getAptosClient } from "../helper";
-import { fundAccounts, multiSignerScriptBytecode, publishTransferPackage, singleSignerScriptBytecode } from "./helper";
+import {
+  createAndFundMultisigAccount,
+  createMultisigTransaction,
+  fundAccounts,
+  multiSignerScriptBytecode,
+  publishTransferPackage,
+  singleSignerScriptBytecode,
+} from "./helper";
 
 describe("transaction simulation", () => {
-  const { aptos } = getAptosClient();
+  const { aptos, config } = getAptosClient();
   const contractPublisherAccount = Account.generate();
   const singleSignerED25519SenderAccount = Account.generate({ scheme: SigningSchemeInput.Ed25519, legacy: false });
   const legacyED25519SenderAccount = Account.generate();
   const singleSignerSecp256k1Account = Account.generate({ scheme: SigningSchemeInput.Secp256k1Ecdsa });
   const recieverAccounts = [Account.generate(), Account.generate()];
   const secondarySignerAccount = Account.generate();
   const feePayerAccount = Account.generate();
+  let multisigAddress: string;
+  const multisigEntryFunction: InputEntryFunctionData = {
```

Comment:
it doesnt seem differrent than what we had before, any reason for this const?
https://github.com/aptos-labs/aptos-ts-sdk/pull/506/files#diff-f97493b76e508e849a876ecb35492119c3ebf540140bda3c1a7b3313a32be2deL528

Reply:
Right, it has the same value as before. It is defined as a `const` to use it consistently in different places. Specifically, the simulation query (https://github.com/aptos-labs/aptos-ts-sdk/pull/506/files#diff-f97493b76e508e849a876ecb35492119c3ebf540140bda3c1a7b3313a32be2deR77) should be consistent to what has been created on-chain (https://github.com/aptos-labs/aptos-ts-sdk/pull/506/files#diff-a4bf74852d27f5fbf001c91e102e84c7f0847b52fcd0e29b9207305bde9abe44R135).